build/grunt.js

```diff
@@ -449,7 +448,6 @@ module.exports = function(grunt) {
     'cssmin',
 
     'copy:fonts',
-    'copy:swf',
```

Comment:
our switch to node 8/npm 5 meant that packages were flattened, so, the swf snuck back into the dist folder.